addons/web/static/src/js/core/registry.js

```diff
@@ -105,8 +106,10 @@ odoo.define("web.Registry", function (require) {
          */
         keys() {
             if (!this._sortedKeys) {
+                const proto = Object.getPrototypeOf(this.map) || {};
+                const keys = [...new Set([...Object.keys(proto), ...Object.keys(this.map)])];
```

Comment:
pretty sure that we should not use this syntax here, it breaks internet explorer